lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -25,11 +25,7 @@ def self.run(params)
 
           # skip_if_exists is deprecated but we want to keep backward compatibility.
           if params[:if_exists].nil?
-            params[:if_exists] = if params[:skip_if_exists].nil? || params[:skip_if_exists] == false
-                                   :fail
-                                 else
-                                   :skip
-                                 end
+            params[:if_exists] = params[:skip_if_exists] ? :skip : :fail
           end
```

Comment:
⛏️  I just realized we could write this in an even more concise way, replacing the `if ….nil?` with the use of `||=`:
```suggestion
          params[:if_exists] ||= params[:skip_if_exists] ? :skip : :fail
```

Reply:
Love it!

Reply:
I wanted to apply the suggestion from GitHub, but it doesn't work...

<img width="750" alt="image" src="https://github.com/wordpress-mobile/release-toolkit/assets/1218433/266943d0-762b-4622-9053-8d68b4f67438">